packages/shipjs/src/step/fetchAndRebase.js

```diff
@@ -0,0 +1,11 @@
+import runStep from './runStep';
+import { run } from '../util';
+
+export default ({ remote, currentBranch, dir, dryRun }) =>
+  runStep({ title: 'Rebasing.' }, () => {
+    run({
+      command: `git fetch && git rebase ${remote}/${currentBranch} ${currentBranch}`,
```

Comment:
I believe `git fetch && git rebase` can be replaced with `git pull --rebase`.

Reply:
This can be done later, but we could add integrations tests for the full flow. We create a fake project in which Ship.js is a dependency (like any of our repos) and perform flows with a mocked CI.

Reply:
I looked a bit further and there's a workflow similar to this in **packages/shipjs-lib/tests** with shell scripts bootstrapping mock repositories. This could be reused for **packages/shipjs** probably.

I won't be able to put more time on it during this sprint, so we can do it later or keep this PR open and add it here next sprint, both are fine for me.

Reply:
Yes! Can you just add a ticket in Jira to track it?